configure.ac

```diff
@@ -2199,6 +2203,15 @@ AS_CASE([$ac_sys_system],
     LIBS="$LIBS -lwasi-emulated-signal -lwasi-emulated-getpid -lwasi-emulated-process-clocks"
     echo "#define _WASI_EMULATED_SIGNAL 1" >> confdefs.h
 
+    AS_VAR_IF([enable_wasm_pthreads], [yes], [
+      # Note: update CFLAGS because ac_compile/ac_link needs this too
+      CFLAGS="$CFLAGS -target wasm32-wasi-threads -pthread"
```

Comment:
This looks fishy; `CFLAGS_NODIST` should be sufficient, no?

Reply:
it wasn't sufficient as far as i tried.

Reply:
i re-investigated this a bit.
without this, configure fails to find pthread_create, sem_init, etc because they are only available in the sysroot for wasm32-wasi-threads.

Reply:
Thanks for investigating. SGTM.

Reply:
> Please note your findings in the comment in line 2207.

done